var/spack/repos/builtin/packages/bash/package.py

```diff
@@ -61,14 +61,21 @@ def determine_version(cls, exe):
     def configure_args(self):
         spec = self.spec
 
-        return [
-            'LIBS=' + spec['ncurses'].libs.link_flags,
+        args = [
             '--with-curses',
             '--enable-readline',
             '--with-installed-readline',
             '--with-libiconv-prefix={0}'.format(spec['iconv'].prefix),
+            'LIBS=' + spec['ncurses'].libs.link_flags,
         ]
 
+        # https://github.com/Homebrew/legacy-homebrew/pull/23234
+        # https://trac.macports.org/ticket/40603
+        if 'platform=darwin' in spec:
+            args.append('CFLAGS=-DSSH_SOURCE_BASHRC')
```

Comment:
Having checked a note in the homebrew comment, I can confirm, that Debian enables this as well on their default bash builds.
So I'd vote for enabling it unconditionally.